_config.yml

```diff
@@ -27,7 +27,9 @@ github_fork_url: "https://github.com/quarkusio/quarkus"
 language: en
 search:
   # The URL of the remote search service
-  url: "https://search.quarkus.io/"
+  script: "/assets/javascript/search-wc.js"
+  script-copy: "https://search.quarkus.io/static/bundle/main.js"
```

Comment:
Shoudn't this rather be `script-to-copy`? It's not itself the copy... ?

Reply:
I've changed to `script-mode=cached/direct`